app/formations/mine/page.tsx

```diff
@@ -25,8 +25,10 @@ export default async function MyFormations() {
 
   return (
     <div className="container">
-      <Link href={`/users/${username}`} className="flex justify-center pb-4">
-        <Button>View your profile</Button>
+      <Link href={`/users/${username}`}>
+        <Button variant={"analytica"} className="mb-4">
```

Comment:
technically you don't need `{}` here.. nit picky, I don't like seeing them unless you need them

Reply:
i only use them b/c that's what my tab complete automatically does xD, not directly needed